README.md

```diff
@@ -211,18 +232,19 @@ This function validates and converts a samplesheet to a ready-to-use Nextflow ch
 
 #### Usage
 
-The function requires two different inputs: the samplesheet and the schema used for the samplesheet. Both files need to be passed through the `file()` function as input for this function.
+The function requires two different inputs: the param used by the user to provide a samplesheet and the schema used for the samplesheet. Both files need to be passed through the `file()` function as input for this function.
 
 ```nextflow
 Channel.validateAndConvertSamplesheet(
-   file('path/to/samplesheet', checkIfExists:true),
+   file(params.input, checkIfExists:true),
    file('path/to/schema', checkIfExists:true)
 )
 ```
 
 For examples on how to process the created channel, see the [examples/](examples/) folder
 
-<!--Note that in order to fully validate the sample sheet you must always run [`validateParameters()`](https://github.com/mirpedrol/nf-validation/blob/ce409583b4582f4221cbf0d0d3917e35f4ba628d/README.md#L116) with the [`schema` field provided](https://github.com/mirpedrol/nf-validation/blob/ce409583b4582f4221cbf0d0d3917e35f4ba628d/README.md#L200).-->
+> **Note**  
+> Note that in order to fully validate the sample sheet you must always run [`validateParameters()`](#validateparameters) with the [`schema` field provided](#validate-an-input-file-provided-by-params-with-another-json-schema).
```

Comment:
Ah ok, so you're saying that this channel factory doesn't validate the samplesheet? Hence you need to run `validateParameters()` to do that?

So the schema is only used in order to know the types required I guess? Maybe I need a refresher on how this all fits together..

Reply:
Ok, after discussion this is clear. We are using the schema to understand what the channel needs (meta fields etc), but here we are _not_ validating the samplesheet file contents against it. I think this docs note needs to mention this to explain why the `validateParameters()` needs to be called.

Alternatively, seeing as we have everything here already, it might be nice to run validation a second time - just in case parameter validation is not being run. Only reason not to do it is if it incurs a significant time penalty. If we take this route I think we can remove this note in the docs.

Would be good to have log debug messages either side of validation so that we can easily track how long it's taking.